<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	font-family: 700 15pt 'Helvetica';
}

QPushButton {
	border-radius: 4px;
	border: 1px solid #999999;
	background-color: rgba(100, 100, 255, 64);
}

QPushButton:hover:!pressed {
	background-color: rgba(100, 100, 255, 128);
	border: 1px solid #cccccc;
}

QPushButton:pressed {
	background-color: qlineargradient(spread:pad, x1:0.507772, y1:0, x2:0.503, y2:1, stop:0 rgba(100, 100, 255, 255), stop:1 rgba(100, 100, 255, 180));
	border: 1px solid #ffffff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="codeInputTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPlainTextEdit {
	background-color: rgba(100, 100, 255, 64);
	border-radius: 4px;
}

QPlainTextEdit:hover {
	background-color: rgba(100, 100, 255, 128);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Enter AQA-ARM assembly code here</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>3</width>
      <height>320</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="runCodeButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QListWidget" name="registerListView">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
	background-color: rgba(100, 100, 255, 64);
	border-radius: 4px;
}

QListWidget:hover {
	background-color: rgba(100, 100, 255, 128);
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="helpLink">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 300 13pt &quot;Helvetica Neue&quot;;
	text-decoration: none;
	color: rgba(255, 255, 255, 128);
}

QLabel:hover {
	color: rgba(255, 255, 255, 255);
}

QLabel:pressed {
	font: 700 13pt &quot;Helvetica Neue&quot;;
}</string>
    </property>
    <property name="text">
     <string>&lt;a href=&quot;https://filestore.aqa.org.uk/resources/computing/AQA-75162-75172-ALI.PDF&quot;&gt;AQA Arm instruction set [link]&lt;/a&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="helpLink_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font: 300 13pt &quot;Helvetica Neue&quot;;
	text-decoration: none;
	color: rgba(255, 255, 255, 128);
}

QLabel:hover {
	color: rgba(255, 255, 255, 255);
}

QLabel:pressed {
	font: 700 13pt &quot;Helvetica Neue&quot;;
}</string>
    </property>
    <property name="text">
     <string>Michael Rowley</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="openExternalLinks">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="commandLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgba(100, 100, 255, 64);
	border-radius: 4px;
}

QLineEdit:hover {
	background-color: rgba(100, 100, 255, 128);
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="executeCommandButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
